### DISCUSSION TITLE
list of data to import

### DISCUSSION THREAD

**Discussion by Cryptodarktrader** [anpigon/obsidian-book-search-plugin#82] [General]
Hello, I know that this addon extracts information from Google Books but I would like to know if anyone has a list or where I can find it of all the fields that can be extracted from a book

Can I configure it to fill in the language of the book or other data that does not appear in the examples?

**Comment by writtenfool** [comment]
The info available from this plugin is https://github.com/anpigon/obsidian-book-search-plugin#template-variables-definitions located  here in the readme.md .

The data available from the Google Books API - I'm guessing this is the source, but it may not be - a glance thru the code in this plugin would answer this ... if the Google Books API is the source, here is the reference for what can be derived https://developers.google.com/books/docs/v1/reference/volumes#resource-representations

**Comment by Cryptodarktrader** [comment]
Until now the definitions indicated are:


title: {{title}}
author: {{author}}
category: {{category}}
description: {{description}}	
publisher: {{publisher}}
totalPage: {{totalPage}}
coverUrl: {{coverUrl}}	
publishDate: {{publishDate:}}	
isbn10: {{isbn10}}	
isbn13: {{isbn30}}


I have tested that it also works (if the book has that information):

subtitle: {{subtitle}}
previewLink: {{previewLink}}

**Reply by jonasgantner** [reply]
Thank you. There is another definition: {{link}}
This returns the link to google books e.g. https://play.google.com/store/books/details?id=fFCjDQAAQBAJ

**Comment by writtenfool** [comment]
If we want other data from Google Books, we'd have to lobby with the developer to update this plugin to include items like Language.
